### PR TITLE
"Update Test Methods" works on filtered test codeunits #9681

### PR DESCRIPTION
**Pull request by lvanvugt**

### CODE REVIEW

File: Modules/DevTools/TestFramework/TestRunner/src/TestSuiteMgt.Codeunit.al [hunk 1]
```diff
@@ -267,7 +267,6 @@ codeunit 130456 "Test Suite Mgt."
         TestRunnerGetMethods: Codeunit "Test Runner - Get Methods";
     begin
         BackupTestMethodLine.Copy(TestMethodLine);
-        TestMethodLine.Reset();
```
Comment (nikolakukrika) [inline Modules/DevTools/TestFramework/TestRunner/src/TestSuiteMgt.Codeunit.al:270]: Hi Luc,

This will not work. We have a bug already: If you put a filter to the individual test method and select UpdateTestMethods action, it will load all methods except that method. That method will disappear.

Also the problem is in the behavior, I would not expect that only filtered list is updated.

Could we do this implementation:
Check if there is a single codeunit, if yes, then update all
If there is multiple, prompt the user to select if they only want to update all (default) or selected? (Something similar to Run Tests action) 
Then we can keep this code as it is and add new path to update selected methods codeunit.

**Review by nikolakukrika — CHANGES_REQUESTED** [review]
See the comment, could we update it to the prompt to make it more discoverable?

File: Modules/DevTools/TestFramework/TestRunner/src/TestSuiteMgt.Codeunit.al (hunk 1 above)
Comment (lvanvugt) [reply Modules/DevTools/TestFramework/TestRunner/src/TestSuiteMgt.Codeunit.al:270]: I am totally fine with your suggestion. Thanx for the effort.

File: Modules/DevTools/BusinessCentralPerformanceToolkit/src/BCPTLogEntry.Table.al [hunk 2]
```diff
@@ -5,6 +5,7 @@
 table 149002 "BCPT Log Entry"
 {
     DataClassification = SystemMetadata;
+    DrillDownPageId = "BCPT Log Entries";
```
Comment (nikolakukrika) [inline Modules/DevTools/BusinessCentralPerformanceToolkit/src/BCPTLogEntry.Table.al:8]: Is this change related?

Comment (lvanvugt) [reply Modules/DevTools/BusinessCentralPerformanceToolkit/src/BCPTLogEntry.Table.al:8]: No, sorry, this should not be in the PR. See #9719. As described there I wanted to change another PR for that, but GitHub seems not to allow me. Or even worse: it incorporates it in the existing PR. Any tips on how to separate PRs are more than welcome.

Comment (stkillen) [reply Modules/DevTools/BusinessCentralPerformanceToolkit/src/BCPTLogEntry.Table.al:8]: Hi @lvanvugt you should look into branches. I responded to your issue.

**Comment by JesperSchulz** [comment]
This is odd! Somehow this PR was merged together with the other one! Something in your branch setup was not quite right it seems!

**Comment by JesperSchulz** [comment]
Based on the code comments though, it seems like this PR should have gotten abandoned anyway :-) I hence won't reopen it. If I'm mistaken, please create a new PR and excuse my mistake!

**Comment by lvanvugt** [comment]
I probably am to to source to all troubles here ;-). Need to learn this.
